sherlock-platform/customization/com.google.sherlock.customization.iml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module relativePaths="true" type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/resources" type="java-resource" />
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+    <orderEntry type="module" module-name="intellij.platform.ide.impl" />
+    <orderEntry type="module" module-name="intellij.idea.customization.base" />
+    <orderEntry type="module" module-name="intellij.platform.experiment" scope="RUNTIME" />
+    <orderEntry type="module" module-name="intellij.platform.resources" />
```

Comment:
Do we need all of these?

Reply:
Not all, but `AnAction` depends on` intellij.platform.ide.impl`. Have removed the others.